recipes/windse/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "windse" %}
+{% set branch = "driver" %}
+{% set version = "0.5" %}
+{% set fenics_version = "2018.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ fenics_version }}_{{ version }}
+
+source:
+
+  url: https://github.com/NREL/WindSE/raw/{{ branch }}/archives/{{ name }}-{{ fenics_version }}_{{ version }}.tar.gz
+  sha256: 0811c220059867c7d35af6366631005f91c6553c39baa54794a70c9bf59b7665
+
+build:
+  number: 0
+  skip: true  # [win]
```

Can this be `noarch`?  I guess you are skipping Windows as a dependency is missing. That would be ok.

```suggestion
  noarch: python
```

---

Some of windse's dependencies are not built for windows, so I have to skip building on windows.

---

As long as the package is building fine on Linux with `noarch`, we can merge with a failing Windows build.

---

Oh! In that case it should be fine. I originally had `noarch: python` but removed it to skip windows when the windows check failed.